src/core/Menu/Menu.tsx

```diff
@@ -23,14 +23,26 @@ export type MenuProps = {
    * and selected item should have `aria-selected={true}`.
    */
   children: React.ReactNode;
+  /**
+   * Should the first selected or enabled menu item be focused automatically?
+   * @default true
+   */
+  bringFocusInside?: boolean;
 } & Omit<CommonProps, 'title'>;
 
 /**
  * Basic menu component. Can be used for select or dropdown components.
  */
 export const Menu = React.forwardRef<HTMLUListElement, MenuProps>(
   (props, ref) => {
-    const { children, role = 'menu', className, style, ...rest } = props;
+    const {
+      children,
+      role = 'menu',
+      bringFocusInside = true,
```

Comment:
In other inputs we have `setFocus`. Maybe use the same name here.

Reply:
Done but it's weird because it does not really mean "set focus on menu"